website/docs/r/managed_redis.html.markdown

```diff
@@ -154,10 +154,14 @@ A `default_database` block supports the following:
 
 * `eviction_policy` - (Optional) Specifies the Redis eviction policy. Possible values are `AllKeysLFU`, `AllKeysLRU`, `AllKeysRandom`, `VolatileLRU`, `VolatileLFU`, `VolatileTTL`, `VolatileRandom` and `NoEviction`. Defaults to `VolatileLRU`.
 
-* `geo_replication_group_name` - (Optional) The name of the geo-replication group. If provided, a geo-replication group will be created for this database with itself as the only member. Use [`azurerm_managed_redis_database_geo_replication`](azurerm_managed_redis_database_geo_replication.html) resource to manage group membership, linking and unlinking. All databases to be linked have to have the same group name. Refer to the [Managed Redis geo-replication documentation](https://learn.microsoft.com/azure/redis/how-to-active-geo-replication) for more information. Changing this forces a new database to be created, data will be lost and Managed Redis will be unavailable during the operation.
+* `geo_replication_group_name` - (Optional) The name of the geo-replication group. If provided, a geo-replication group will be created for this database with itself as the only member. Use [`azurerm_managed_redis_database_geo_replication`](azurerm_managed_redis_database_geo_replication.html) resource to manage group membership, linking and unlinking. All databases to be linked have to have the same group name. Conflicts with `persistence_append_only_file_backup_frequency_in_seconds` and `persistence_redis_database_backup_frequency_in_hours`, persistence can only be enabled on non-geo-replicated databases. Refer to the [Managed Redis geo-replication documentation](https://learn.microsoft.com/azure/redis/how-to-active-geo-replication) for more information. Changing this forces a new database to be created, data will be lost and Managed Redis will be unavailable during the operation.
 
 * `module` - (Optional) A `module` block as defined below. Refer to [the modules documentation](https://learn.microsoft.com/azure/redis/redis-modules) to learn more.
 
+* `persistence_append_only_file_backup_frequency_in_seconds` - (Optional) The frequency of Append Only File (AOF) backups. Possible value is `1`. Providing this value implies AOF persistence method is enabled. Conflicts with `persistence_redis_database_backup_frequency_in_hours`, only one persistence method is allowed. Conflicts with `geo_replication_group_name`, persistence can only be enabled on non-geo-replicated databases. Refer to [the persistence documentation](https://learn.microsoft.com/azure/redis/how-to-persistence) to learn more.
```

Comment:
The enum exposed is `1s`, not `1`?

Reply:
The "exception" here is that the property was unfortunately misnamed, since the values are `Durations` not numerical values. I'll check the docs PR asap to make sure that's clear, but thanks for following that up 👍